### ISSUE TITLE
Equipments not showing up

### ISSUE THREAD

**Issue by graaaeee** [Uuvana-Studios/longvinter-windows-client#926]
I cannot see the equipment already equipped. I have an assault rifle equipped and telescopic fishing rod in my backpack but I can't seem to see them. I also have the tool for directions equipped.

I expect my equipments to be seen again.

**Desktop (please complete the following information):**
Windows 10
Longvinter Beta 1.0.7 (Steam ver.)

![bug](https://user-images.githubusercontent.com/107413777/173364634-3109c55a-8108-4612-bee9-750339bc5483.jpg)

**Comment by faketucker** [comment]
Yes, this does happen sometimes. Not sure what causes it. It could be because of server lag.
